setuptools_odoo/core.py

```diff
@@ -12,50 +12,68 @@
 from .manifest import read_manifest, is_installable_addon
 from .git_postversion import get_git_postversion
 
-LEGACY_MODE = os.environ.get('SETUPTOOLS_ODOO_LEGACY_MODE')
-
-if LEGACY_MODE:
-    warn('SETUPTOOLS_ODOO_LEGACY_MODE support will be removed in '
-         'setuptools-odoo 1.1.0. Please switch to the '
-         'new package naming scheme.')
-
 ODOO_VERSION_INFO = {
     '7.0': {
         'odoo_dep': 'openerp>=7.0a,<8.0a',
         'base_addons': base_addons.openerp7,
-        'addon_dep_version': '' if not LEGACY_MODE else '>=7.0a,<8.0a',
-        'pkg_name_pfx': ('openerp7-addon-'
-                         if not LEGACY_MODE else 'openerp-addon-'),
+        'pkg_name_pfx': 'openerp7-addon-',
         'addons_ns': 'openerp_addons',
+        'namespace_packages': ['openerp_addons'],
     },
     '8.0': {
         'odoo_dep': 'odoo>=8.0a,<9.0a',
         'base_addons': base_addons.odoo8,
-        'addon_dep_version': '' if not LEGACY_MODE else '>=8.0a,<9.0a',
-        'pkg_name_pfx': ('odoo8-addon-'
-                         if not LEGACY_MODE else 'odoo-addon-'),
+        'pkg_name_pfx': 'odoo8-addon-',
         'addons_ns': 'odoo_addons',
+        'namespace_packages': ['odoo_addons'],
     },
     '9.0': {
         'odoo_dep': 'odoo>=9.0a,<9.1a',
         'base_addons': base_addons.odoo9,
-        'addon_dep_version': '' if not LEGACY_MODE else '>=9.0a,<9.1a',
-        'pkg_name_pfx': ('odoo9-addon-'
-                         if not LEGACY_MODE else 'odoo-addon-'),
+        'pkg_name_pfx': 'odoo9-addon-',
         'addons_ns': 'odoo_addons',
+        'namespace_packages': ['odoo_addons'],
     },
     '10.0': {
         'odoo_dep': 'odoo>=10.0,<10.1dev',
         'base_addons': base_addons.odoo10,
-        'addon_dep_version': '',
         'pkg_name_pfx': 'odoo10-addon-',
         'addons_ns': 'odoo.addons',
+        'namespace_packages': ['odoo', 'odoo.addons'],
+    },
+    '11.0': {
+        'odoo_dep': 'odoo>=11.0a,<10.1dev',
```

Comment:
Maybe the dev version number isn't available yet but 10.1dev isn't correct here.

Reply:
Indeed. Thanks for reporting.

Note this is still very experimental as we've not agreed yet with Odoo on how best to support this.

Reply:
Fixed and rebased.